src/components/EntityMap.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <MglMap
+    :zoom="10"
+    :accessToken="accessToken"
+    :mapStyle="mapStyle"
+    :center="coordinatesList[0]"
+    @load="onMapLoaded"
+  >
+    <MglMarker
+      v-for="(coordinates, index) in coordinatesList"
+      :key="index"
+      :coordinates="coordinates"
+    />
+    <MglNavigationControl position="top-right" />
+  </MglMap>
+</template>
+
+<script>
+  import Mapbox from 'mapbox-gl';
+  import { MglMap, MglMarker, MglNavigationControl } from 'vue-mapbox';
+
+  const accessToken =
```

Comment:
Not a blocker now, but I still think eventually we want this accessed via some other config (e.g. $scaife.config):

![](https://github.trello.services/images/mini-trello-icon.png) [Remove hardcoded Mapbox API token and document configuration](https://trello.com/c/H7oh2zay)

Reply:
I can move this once we get a new release of Scaife-skeleton out.  See https://github.com/scaife-viewer/scaife-skeleton/pull/57

Reply:
@paltman I've merged and released if you want to update to `0.10.0` --> https://www.npmjs.com/package/scaife-skeleton